internal/resto/structs.go

```diff
@@ -0,0 +1,36 @@
+package resto
```

Comment:
I'd rename the file. `structs` is non descriptive.

Reply:
I rename this file to the `response`. Do you have any better ideas?